vividus-plugin-mobile-app/src/main/java/org/vividus/selenium/mobileapp/MobileAppWebDriverManager.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.selenium.mobileapp;
+
+import java.util.Map;
+import java.util.Optional;
+
+import org.vividus.selenium.IWebDriverProvider;
+import org.vividus.selenium.manager.GenericWebDriverManager;
+import org.vividus.selenium.manager.IWebDriverManagerContext;
+
+import io.appium.java_client.CommandExecutionHelper;
+import io.appium.java_client.HasSessionDetails;
+import io.appium.java_client.android.AndroidMobileCommandHelper;
+import io.appium.java_client.android.HasAndroidDeviceDetails;
+
+public class MobileAppWebDriverManager extends GenericWebDriverManager
+{
+    public MobileAppWebDriverManager(IWebDriverProvider webDriverProvider,
+            IWebDriverManagerContext webDriverManagerContext)
+    {
+        super(webDriverProvider, webDriverManagerContext);
+    }
+
+    public int getStatusBarSize()
+    {
+        return  super.isIOSNativeApp() ? getIOsStatusBar() : getAndroidStatusBar();
+    }
+
+    private int getAndroidStatusBar()
+    {
+        HasAndroidDeviceDetails details = getWebDriverProvider().getUnwrapped(HasAndroidDeviceDetails.class);
+        // https://github.com/appium/java-client/commit/9020174c578bed5e03c24b43c2c5ba590f663201
+        Map<String, Map<String, Object>>  systemBars = CommandExecutionHelper.execute(details,
+                AndroidMobileCommandHelper.getSystemBarsCommand());
+        return Optional.ofNullable(systemBars)
+                       .map(b -> b.get("statusBar"))
+                       .map(sb -> sb.get("height"))
+                       .map(Long.class::cast)
+                       .map(Long::intValue)
+                       .orElse(0);
+    }
+
+    private int getIOsStatusBar()
+    {
+        HasSessionDetails details = getWebDriverProvider().getUnwrapped(HasSessionDetails.class);
+        return ((Long) details.getSessionDetail("statBarHeight")).intValue();
```

Comment:
`statBarHeight` - is that correct?

Reply:
https://github.com/appium/appium-ios-driver/blob/9755e309af6f5cbd4a895ca3f157650a943eff5b/lib/driver.js#L233